tests/Regression/GitHub/2867-data-provider-keys.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+GH-2867: test names show numeric datasource only for iterators as dataprovider
+GH-2639: Second `yield from` is not called from a data provider
+--FILE--
+<?php
+$_SERVER['argv'][1] = '--no-configuration';
+$_SERVER['argv'][1] = '--debug';
```

Comment:
using `phpunit --debug` to show the datasource names is just a suggestion.
Alternatives:
 * let tests fail
 * let them report skip and use `phpunit -v`